### ISSUE TITLE
Netlog stopped working for Firefox

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/browsertime#2202]
Hmm, it seems that the net log for Firefox in Browsertime stopped working. It just creates empty files:

<img width="914" alt="Screenshot 2024-11-09 at 17 28 04" src="https://github.com/user-attachments/assets/a0f91100-11c2-431a-bc85-bb733299edbd">

I tried with 132 to debug the 132 issue.Let me see if the log works locally on my machine.

**Comment by soulgalore** [comment]
It works locally (even though it's confusing it creates so many files). Let me try again on the test server.

**Comment by soulgalore** [comment]
Inside Docker the moz HTTP file is not created. It works if I run directly on my Mac. Let's test on Linux without using Docker.

**Comment by soulgalore** [comment]
I tried on Ubuntu 22.04.4 LTS (Mozilla Firefox 132.0.1) and the netlog files are just empty.

**Comment by soulgalore** [comment]
Created https://bugzilla.mozilla.org/show_bug.cgi?id=1930592

**Comment by soulgalore** [comment]
Is this something you have seen @gmierz ?

**Comment by gmierz** [comment]
No, I haven't seen this before but I also haven't used these files before
